lib/rules/await-async-utils.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const { getDocsUrl, ASYNC_UTILS } = require('../utils');
+
+const VALID_PARENTS = [
+  'AwaitExpression',
+  'ArrowFunctionExpression',
+  'ReturnStatement',
+];
+
+const ASYNC_UTILS_REGEXP = new RegExp(`^(${ASYNC_UTILS.join('|')})$`);
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Enforce async utils to be awaited properly',
+      category: 'Best Practices',
+      recommended: true,
+      url: getDocsUrl('await-async-utils'),
+    },
+    messages: {
+      awaitAsyncUtil: 'Promise returned from `{{ name }}` must be handled',
+    },
+    fixable: null,
+    schema: [],
+  },
+
+  create: function(context) {
+    const testingLibraryUtilUsage = [];
+    return {
+      [`CallExpression > Identifier[name=${ASYNC_UTILS_REGEXP}]`](node) {
+        if (!isAwaited(node.parent.parent) && !isPromiseResolved(node)) {
+          testingLibraryUtilUsage.push(node);
+        }
+      },
+      'Program:exit'() {
```

Comment:
Ah interesting, I didn’t know about this

Reply:
Me either! I learnt about it thanks to @thomlom, you can find more info [here](https://eslint.org/docs/developer-guide/selectors#listening-for-selectors-in-rules)